ovos_utils/system.py

```diff
@@ -56,6 +56,15 @@ def ntp_sync():
     subprocess.call('ntpd -gq', shell=True)
     subprocess.call('service ntp start', shell=True)
 
+def timesync_sync(sudo=True):
```

Comment:
Please add unit test coverage; shutdown or reboot tests can be mostly copy/pasted with a minor change to the expected subprocess call